completion_aggregator/api/v1/views.py

```diff
@@ -361,7 +362,10 @@ def _parse_aggregator(self, course_key, params=None):
         """
         Handles fetching and return aggregator data, regardless of the method used
         """
-        course_key = CourseKey.from_string(course_key)
+        try:
+            course_key = CourseKey.from_string(course_key)
+        except InvalidKeyError:
+            raise NotFound("Invalid block key: '{}'.".format(course_key))
```

Comment:
Nit: it's "course key".

Btw. could you please also add a message to line 384 to have consistent errors? It could be something along:
```python
raise NotFound("User '{}' does not have an active enrollment in course '{}'.".format(self.user, course_key))
```